### PR TITLE
Update WCF to Take Dependency on NetStandard2.0

### LINKED ISSUE #1726: Update WCF to Take Dependency on NetStandard2.0
WCF needs to take dependency on NetStandard2.0 to use any new added APIs. 

This is blocking https://github.com/dotnet/wcf/issues/702.

### PR DESCRIPTION
**Pull request by shmao**
Fix #1726.

/cc: @mconnew @zhenlan @hongdai

### CODE REVIEW

File: src/System.Private.ServiceModel/src/System/ServiceModel/Description/CustomAttributeProvider.cs [hunk 1]
```diff
@@ -87,9 +87,15 @@ public object[] GetCustomAttributes(Type attributeType, bool inherit)
                     return this.MemberInfo.GetCustomAttributes(attributeType, inherit).ToArray();
                 case AttributeProviderType.ParameterInfo:
                     //GetCustomAttributes could return null instead of empty collection for a known System.Relection issue, workaround the issue by explicitly checking the null
+#if PreNETStandard17
                     IEnumerable<Attribute> customAttributes = null;
                     customAttributes = this.ParameterInfo.GetCustomAttributes(attributeType, inherit);
                     return customAttributes == null ? null : customAttributes.ToArray();
+#else
+
```
Comment (shmao) [inline src/System.Private.ServiceModel/src/System/ServiceModel/Description/CustomAttributeProvider.cs]: remove this line.

**Comment by shmao** [comment]
Hi @ericstj , can you please have a look at the changes? We are making System.Private.ServiceModel to take dependency on NetStandard2.0 so that it can use new APIs.

Do we need to make changes in contract assemblies and make them to target 1.7?

**Comment by ericstj** [comment]
It looks to me like you are only changing internals and no public surface area.  In that case you don't need to update your reference assemblies.

I noticed no change to your builds file, so this means you're repurposing your default (netstandard1.3) build to target netstandard1.7.  That means your previous netstandard1.3 will no longer build and the dll will copied from your last shipped package.  You'll want to confirm that this is what you expect to do.

**Comment by shmao** [comment]
@ericstj Thanks for reviewing the changes. Yes, our current decision is to repurose the default build to target netstatndard1.7. And we will no longer build NetStandard1.3 dll.

**Comment by shmao** [comment]
@dotnet-bot test OuterLoop please

**Comment by shmao** [comment]
@dotnet-bot test this please

**Comment by StephenBonikowsky** [comment]
Ignore the failing test, nothing to do with you.
LGTM

**Comment by shmao** [comment]
Thanks for the confirmation, @StephenBonikowsky. I'm merging this.

**Pull request merged**